### PR TITLE
Add ToC

### PR DESCRIPTION
**Pull request by apeltzer**
For publications

### CODE REVIEW

File: markdown/publications.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+- [The nf-core project](#the-nf-core-project)
+- [Pipeline publications](#pipeline-publications)
+    - [nf-core/ampliseq](#nf-coreampliseq)
+    - [nf-core/coproid](#nf-corecoproid)
+    - [nf-core/diaproteomics](#nf-corediaproteomics)
+    - [nf-core/eager](#nf-coreeager)
+    - [nf-core/lncpipe](#nf-corelncpipe)
+    - [nf-core/mhcquant](#nf-coremhcquant)
+    - [nf-core/sarek](#nf-coresarek)
 # The nf-core project
```
Comment (apeltzer) [inline markdown/publications.md:16]:
```suggestion
    - [nf-core/sarek](#nf-coresarek)
    
# The nf-core project
```

File: markdown/publications.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 Do you know of an nf-core publication that we're missing?
 Please [make a pull-request](https://github.com/nf-core/nf-co.re/blob/master/markdown/publications.md).
 
+# The nf-core publications
+
+- [The nf-core publications](#the-nf-core-publications)
+- [The nf-core project](#the-nf-core-project)
+- [Pipeline publications](#pipeline-publications)
+    - [nf-core/ampliseq](#nf-coreampliseq)
```
Comment (MaxUlysse) [inline markdown/publications.md:9]:
```suggestion
  - [nf-core/ampliseq](#nf-coreampliseq)
```

File: markdown/publications.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+    - [nf-core/coproid](#nf-corecoproid)
```
Comment (MaxUlysse) [inline markdown/publications.md:10]:
```suggestion
  - [nf-core/coproid](#nf-corecoproid)
```

File: markdown/publications.md [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+    - [nf-core/diaproteomics](#nf-corediaproteomics)
```
Comment (MaxUlysse) [inline markdown/publications.md:11]:
```suggestion
  - [nf-core/diaproteomics](#nf-corediaproteomics)
```

File: markdown/publications.md [hunk 5]
```diff
@@ -1,6 +1,19 @@
...
+    - [nf-core/eager](#nf-coreeager)
```
Comment (MaxUlysse) [inline markdown/publications.md:12]:
```suggestion
  - [nf-core/eager](#nf-coreeager)
```

File: markdown/publications.md [hunk 6]
```diff
@@ -1,6 +1,19 @@
...
+    - [nf-core/lncpipe](#nf-corelncpipe)
+    - [nf-core/mhcquant](#nf-coremhcquant)
+    - [nf-core/sarek](#nf-coresarek)
+
```
Comment (MaxUlysse) [inline markdown/publications.md:16]:
```suggestion
  - [nf-core/lncpipe](#nf-corelncpipe)
  - [nf-core/mhcquant](#nf-coremhcquant)
  - [nf-core/sarek](#nf-coresarek)

```
Sorry for not multilining everything...

**Comment by mashehu** [comment]
Maybe we can use the fancier sticky side ToC like we have in the pipeline specific docs, here as well. I will give it a shot.

**Pull request closed without merging**